src/Membership/MembershipRequestHandler.php

```diff
@@ -48,38 +49,62 @@ public function __construct(
         $this->committeeManager = $committeeManager;
     }
 
-    public function handle(MembershipRequest $membershipRequest)
+    public function registerAsUser(MembershipRequest $membershipRequest): Adherent
     {
         $adherent = $this->adherentFactory->createFromMembershipRequest($membershipRequest);
+        $this->manager->persist($adherent);
+        $this->sendEmailValidation($adherent);
+
+        $this->dispatcher->dispatch(UserEvents::USER_CREATED, new UserEvent($adherent));
+
+        return $adherent;
+    }
+
+    public function sendEmailValidation(Adherent $adherent): void
+    {
         $token = AdherentActivationToken::generate($adherent);
 
-        $this->manager->persist($adherent);
         $this->manager->persist($token);
         $this->manager->flush();
 
         $activationUrl = $this->generateMembershipActivationUrl($adherent, $token);
         $this->mailer->sendMessage(AdherentAccountActivationMessage::createFromAdherent($adherent, $activationUrl));
+    }
+
+    public function registerAsAdherent(MembershipRequest $membershipRequest): void
+    {
+        $adherent = $this->adherentFactory->createFromMembershipRequest($membershipRequest);
+        $adherent->setComMobile($membershipRequest->comEmail);
```

Comment:
I think you need to do this with your FormType on PRE_SUBMIT event:
```
// FormType::preSubmitCallback
$data['comMobile'] = $data['comEmail'];
$event->setData($data);
```

Reply:
POST_SUBMIT event i think